{"name": "test_verify_login_with_valid_credentials", "status": "passed", "start": 1722048271987, "stop": 1722048272884, "uuid": "2081ea0a-28d5-4a1c-b6b8-d35f21313c5f", "historyId": "14f2072526db18090e3cb1e2c1ce29e3", "testCaseId": "14f2072526db18090e3cb1e2c1ce29e3", "fullName": "test.test_Login.TestLogin#test_verify_login_with_valid_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Raunak"}, {"name": "thread", "value": "2884-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_Login"}]}